src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs

```diff
@@ -193,6 +193,34 @@ private ReadOnlySpan<char> PartialUtf16CodePoint
             }
         }
 
+        /// <summary>
+        /// The partial base64 code point.
+        /// </summary>
+        private ReadOnlySpan<byte> PartialBase64CodePoint
+        {
+            get
+            {
+                Debug.Assert(PreviousSegmentEncoding == SegmentEncoding.Base64);
+
+                ReadOnlySpan<byte> partialCodePointBytes = PartialCodePointRaw;
```

Comment:
Similarly, `PartialCodePointRaw` should be renamed to something like `PartialStringDataRaw` to reflect that it's more general-purpose than Unicode code points.